watertap/tools/parameter_sweep/tests/test_parameter_sweep.py

```diff
@@ -1653,7 +1653,10 @@ def _build_header_list_from_csv(csv_filename):
 
 def _read_output_h5(filepath):
 
-    f = h5py.File(filepath, "r")
+    if type(filepath) is str:
```

Comment:
```suggestion
    if isinstance(filepath, str):
```
Slightly more general: allows for classes which inherit from string.